plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotLatestNoGrpcConnectorSmokeTest.java

```diff
@@ -27,12 +25,6 @@ protected boolean isSecured()
         return false;
     }
 
-    @Override
-    protected String getPinotImageName()
```

Comment:
The method in BasePinotConnectorSmokeTest is redundant now.

Reply:
Sure. Let me drop it.